components/staticFeeds/sample-widget_message.json

```diff
@@ -0,0 +1,10 @@
+{
+  "mesg": "Success",
+  "result": [
+    {
+      "longMessage": "Your account will be deactivated soon after some time.",
+      "shortMessage": "Your account will be deactivated soon after some time."
```

Comment:
Might be best for the sample to differentiate `longMessage` and `shortMessage` so it more effectively demonstrates how the two message lengths are used differently.

Reply:
Good call. +1